src/semgrep_agent/meta.py

```diff
@@ -35,13 +35,18 @@ def event_name(self) -> str:
 
     @cachedproperty
     def repo(self) -> gitpython.Repo:  # type: ignore
-        repo = gitpython.Repo()
+        repo = gitpython.Repo(".", search_parent_directories=True)
         debug_echo(f"found repo: {repo!r}")
         return repo
 
     @cachedproperty
     def repo_name(self) -> str:
-        return Path.cwd().name
+        repo = gitpython.Repo(".", search_parent_directories=True)
```

Comment:
Do we want to traverse up or should we just let users always call semgrep action at repo root?

Reply:
🤷‍♀️ I feel like it is nice to be able to call from anywhere inside your repo